lib/open_api/config.ex

```diff
@@ -261,6 +263,8 @@ defmodule OpenAPI.Config do
     :ignore,
     :merge,
     :operation_location,
+    :operation_use_tags,
+    :operation_default_module,
```

Comment:
This is very picky of me, but would you mind alphabetizing these?

Reply:
sure, done also in all the places (typespec, module init, private functions helpers)